airflow/contrib/hooks/gcp_vision_hook.py

```diff
@@ -116,6 +136,11 @@ def get_conn(self):
 
     @cached_property
     def annotator_client(self):
+        """
+        Creates ImageAnnotatorClient.
```

Comment:
```suggestion
        Creates ImageAnnotatorClient.
        
```

Empty line is required after description.